website/i18n/ar/docusaurus-plugin-content-docs/current/DriverBinaries.md

```diff
@@ -15,7 +15,7 @@ If you are looking to set up mobile and other UI platforms, have a look into our
 
 ## Chromedriver
 
-To automate Chrome you can download Chromedriver directly on the [project website](http://chromedriver.chromium.org/downloads) or through the NPM package:
```

Comment:
I don't think this should be replaced with a page that leads to Edgedriver.

Reply:
my bad. Fixed